website/content/docs/agent/telemetry.mdx

```diff
@@ -89,6 +89,17 @@ These are some metrics emitted that can help you understand the health of your c
 
 **What to look for:** For a healthy cluster, you're looking for a `lastContact` lower than 200ms, `leader` > 0 and `candidate` == 0. Deviations from this might indicate flapping leadership.
 
+### Certificate authority expiration
+
+| Metric Name                            | Description                                                             | Unit    | Type  |
+| :------------------------- | :---------------------------------------------------------------------------------- | :------ | :---- |
+| `consul.mesh.active-root-ca.expiry` | The number of seconds until the root CA expires. Consul updates the value every hour. | seconds | gauge |
+| `consul.mesh.active-signing-ca.expiry` | The number of seconds until the signing CA expires. Consul updates the value every hour. | seconds | gauge |
+
+** Why they're important:** When using an external system, such as Vault, to manage the CA, you should monitor renewals to ensure that certificates do not expire unexpectedly and so that you can rotate as necessary. 
```

Comment:
Don't you always need to monitor `consul.mesh.active-root-ca.expiry` no matter what, since Consul will never rotate that on your behalf (whether or not you're using the Vault provider)?

It also seems like you'd want to monitor the signing CA no matter what, just in case the automatic renewal fails for whatever reason, which is more likely if using an external provider (such as Vault).

Reply:
I was under the impression, mistakenly it seems, that the built-in CA handled rotation for you. IE. that you only needed to worry about rotation when using an external provider.

I'll rework this with that in light.